src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(),(int)(Math.random() * 101));

```

Comment:
let's create a constant for 101 number https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names